compiler/x/env/OMRCPU.cpp

```diff
@@ -26,11 +26,14 @@
 #include "env/ProcessorInfo.hpp"
 #include "x/runtime/X86Runtime.hpp"
 
+
 TR_X86CPUIDBuffer *
-OMR::X86::CPU::queryX86TargetCPUID(TR::Compilation *comp)
+OMR::X86::CPU::queryX86TargetCPUID()
    {
 #ifdef J9_PROJECT_SPECIFIC
-   return NULL;
+   static TR_X86CPUIDBuffer buf = { {'U','n','k','n','o','w','n','B','r','a','n','d'} };
+   jitGetCPUID(&buf);
+   return &buf;
```

Comment:
This change does not seem directly related to removing the comp parameter. Why is this J9 specific code provided here rather than in the OpenJ9 project (https://github.com/eclipse/openj9/blob/master/runtime/compiler/trj9/x/env/J9CPU.cpp) ? It seems like there's an opportunity to clean up this particular function...